packages/maestro/src/plugin/twig.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * This is a transferred version of `larowlan/vite-plugin-twig-drupal
+ *
+ * TODO: reimplement with
+ *   - more typesafe approach
+ *   - remove react support
+ *   - decompose
+ */
+
+import { resolve, dirname } from "node:path";
+import { existsSync } from "node:fs";
+
+import Twig from "twig";
```

Comment:
I am concerned that this implementation of twig will behave differently from the twig implementation used in the PHP ecosystem. Already it notes that a few things that behave differently - https://github.com/twigjs/twig.js/wiki/Implementation-Notes#differences--implementation-notes

We had this risk before with wingsuit as well I think since it also leveraged a node implementation of twig.

Reply:
[Context](https://github.com/international-labour-organization/designsystem/issues/1046#issuecomment-2169653852)

I get the point, but it's the best we have in the `node` ecosystem. What's your alternative?

Reply:
That is why I suggested switching to using the UI patterns module in Drupal as our dev playground. With the docker image, running it is trivial and already the docker image is being used to run the twig tests so it is not a new addition.

But as we previously discussed, I don't know whether hot reloading would work and that environment is not as interactive as storybook. So this is a risk we have to live with I guess.